src/components/sideBar/sideBar.tsx

```diff
@@ -22,11 +22,25 @@ import { FiMenu } from "react-icons/fi";
 import { mainSections } from "../../siteMap/mainSections";
 import MenuItem from "./menuItem/menuItem";
 import { SubsectionMenuItemType } from "./menuItem/types";
+import { useAuthContext } from "../../lib/authentication/authenticationContext";
+import { LoginStatus } from "../../lib/authentication/authTypes";
 
 //#endregion
 
 export default function SimpleSidebar() {
   const { isOpen, onOpen, onClose } = useDisclosure();
+
+  //With the useAuthContext hook you can get the user's authentication status and hide\show the menu items accordingly
```

Comment:
promote the use library components like <IfAuth><Stuff /></IfAuth> and similar for permissions <HasPermission p="write"><Stuff /></HasPermission> instead of using Hooks directly.

Reply:
the commented code was obsolete i upgraded it with

  const { context, isLogged } = useAuthContext();